views/includes/header.hbs

```diff
@@ -15,10 +15,11 @@
 <div class="navbar navbar-default">
     <div class="container">
         <div class="navbar-header">
-            <div class="navbar-brand">Gulliver</div>
+            <div class="navbar-brand"><a href="/">Gulliver</a></div>
         </div>
         <ul class="nav navbar-nav">
-            <li><a href="/pwas">PWAs</a></li>
+            <li><button id="login" type="button" class="btn btn-default btn-sm" disabled>Login</button></li>
```

Comment:
Not to be decided in this PR, but we need to figure out how the login flow fits in the UX. Maybe they will be slightly hidden since most users wont login. But it needs to make sense as part of the "Add a PWA flow".